.github/workflows/static-analysis.yml

```diff
@@ -121,12 +121,34 @@ jobs:
                 with:
                     composer-options: --ansi --prefer-dist
 
-            -   name: Setup REDAXO
+            -   name: Init database
                 run: |
                     sudo /etc/init.d/mysql start
                     mysql -uroot -h127.0.0.1 -proot -e 'create database redaxo5;'
                     git apply .github/workflows/default.config.yml.github-action.diff
+
+            -   name: Setup
+                run: |
                     php .tools/bin/setup
+            -   name: Create user, update config
+                run: |
+                    php redaxo/bin/console user:create myusername mypassword --admin --ansi
+                    php redaxo/bin/console config:set error_email 'test@redaxo.invalid' --ansi
+            -   name: Import mock data
+                run: |
+                    mkdir -p $GITHUB_WORKSPACE/redaxo/data/addons/backup && cp -r .github/imports/inital-content-for-visual-tests.sql $GITHUB_WORKSPACE/redaxo/data/addons/backup/
+                    cp -r .github/imports/media/* $GITHUB_WORKSPACE/media/
+                    cp -f .github/imports/README.md $GITHUB_WORKSPACE/redaxo/src/addons/project/README.de.md
+                    php redaxo/bin/console setup:run -n --lang=de_de --agree-license --db-password=root --db-setup=import --db-import=inital-content-for-visual-tests --admin-username=myusername --admin-password=mypassword --ansi
+            -   name: Install packages
+                run: |
+                    php redaxo/bin/console package:install phpmailer --ansi
+                    php redaxo/bin/console package:install cronjob --ansi
+                    php redaxo/bin/console package:install cronjob/article_status --ansi
+                    php redaxo/bin/console package:install cronjob/optimize_tables --ansi
+                    php redaxo/bin/console package:install debug --ansi
+                    php redaxo/bin/console package:install structure/history --ansi
+                    php redaxo/bin/console package:install structure/version --ansi
```

Comment:
alle packages installieren, damit das db schema vollständig ist

Reply:
Bedeutet das, dass man zukünftig auch lokal beim Ausführen von phpstan eine bestimmte DB-Konstellation braucht?

Reply:
nicht zwangsweise, siehe https://github.com/staabm/phpstan-dba#record-and-replay